tests/functional/i/invalid/invalid_name.py

```diff
@@ -59,12 +59,12 @@ def wrapper(*args, **kwargs):
 
 
 @dummy_decorator(1, [
-    0  # Fixing #119 should make this go away
-# C0103 always points here - line 61  # [invalid-name]
+    0
+
 
 
 ])
-def a_very_very_very_long_function_name_WithCamelCase_to_make_it_sad():  # Should be line 65
+def a_very_very_very_long_function_name_WithCamelCase_to_make_it_sad():  # [invalid-name]

```

Comment:
This actually fixes the long standing issue that is mentioned in the comment here.

Reply:
#119 is closed as 'won't fix' but apparently it was not specific to python 3.6 ?

Reply:
I think it's a combination of `parser` changes and how we do `tolineno` right now (as in, without `end_lineno`). In any case, this change clearly fixes it 😄